src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java

```diff
@@ -81,6 +87,62 @@ public CaseAssignmentResponse assignAccessWithinOrganisation(
             @Valid @RequestBody CaseAssignmentRequest requestPayload) {
         CaseAssignment caseAssignment = mapper.map(requestPayload, CaseAssignment.class);
         List<String> roles = caseAssignmentService.assignCaseAccess(caseAssignment);
-        return new CaseAssignmentResponse(String.format(MESSAGE, StringUtils.join(roles, ',')));
+        return new CaseAssignmentResponse(String.format(ASSIGN_ACCESS_MESSAGE, StringUtils.join(roles, ',')));
+    }
+
+    @GetMapping(path = "/case-assignments", produces = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get Case Assignments in My Organisation", notes = "Get Assignments in My Organisation")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "Case-User-Role assignments returned successfully.",
+                    response = GetCaseAssignmentsResponse.class,
+                    examples = @Example({
+                            @ExampleProperty(
+                                    value = "[\n"
+                                            + "  {\n"
+                                            + "    \"case_id\": \"1588234985453946\",\n"
+                                            + "    \"case_title\": \"Paul Saddlebrook\",\n"
+                                            + "    \"shared_with\": [\n"
+                                            + "      {\n"
+                                            + "        \"idam_id\": \"221a2877-e1ab-4dc4-a9ff-f9424ad58738\",\n"
+                                            + "        \"first_name\": \"Bill\",\n"
+                                            + "        \"last_name\": \"Roberts\",\n"
+                                            + "        \"email\": \"bill.roberts@greatbrsolicitors.co.uk\",\n"
+                                            + "        \"case_roles\": [\n"
+                                            + "          \"[Collaborator]\",\n"
+                                            + "          \"[Creator]\"\n"
+                                            + "        ]\n"
+                                            + "      }\n"
+                                            + "    ]\n"
+                                            + "  }\n"
+                                            + "]",
```

Comment:
Is this example rendered as the correct expected response in Swagger? Looks like the response is an object where the array is within `case_assignments`?

Reply:
Well, I missed that point, thank you Dan. Perhaps generating the JSON String from a constant object with set attributes could be a better way to maintain complex example values like this.

Reply:
Good catch Dan. Thanks! Pushed updated doc. 
Mutlu - Swagger accepts a String value only for that @Example field.

Reply:
@smathangi  - Of course, but what I mean was - we could get that String from a static method call that serialises a constant example object, couldn't we?

Reply:
Ahh, that's an annotation parameter, which would accept only a constant value. A programmatic solution is still possible, but with a little dirty work-around, and not worth discussing in this particular context. Ok, ignore my comment @smathangi .